domain/library/src/main/java/com/into/websoso/domain/library/model/AttractivePoints.kt

```
@@ -0,0 +1,19 @@
+) {
+    WORLDVIEW("세계관", "worldview"),
+    MATERIAL("소재", "material"),
+    CHARACTER("캐릭터", "character"),
+    RELATIONSHIP("관계", "relationship"),
+    VIBE("분위기", "vibe"),
+    ;
+
+    companion object {
+        fun valueOf(name: String): AttractivePoints =
+            AttractivePoints.entries.find { attractivePoints -> attractivePoints.name == name }
+                ?: throw IllegalArgumentException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)